robottelo/ui/locators/base.py

```diff
@@ -1754,22 +1758,23 @@
         "//button[contains(@ng-show, 'list') and "
         "contains(@ng-click, 'removeRepositories')]"),
     "contentviews.repo_search": (
-        By.XPATH, "//input[@ng-model='repositorySearch']"),
+        By.XPATH, "//input[@ng-model='table.searchTerm']"),
```

Comment:
Heh, wanted to propose to push required changes by myself to your branch if that's a big deal - this way we wouldn't have to reject/close your PR. But separate issue to track it for all the entities is better option, of course 👍